src/java.base/share/classes/java/nio/Buffer.java

```diff
@@ -778,6 +778,13 @@ final void checkSession() {
         // setup access to this package in SharedSecrets
         SharedSecrets.setJavaNioAccess(
             new JavaNioAccess() {
+
+                // We are not using a lambda here because this would create a circular dependency with lambda factories.
+                static final JavaNioAccess.SessionAcquisition NO_OP_CLOSE = new JavaNioAccess.SessionAcquisition() {
+                    @Override
+                    public void close() throws RuntimeException {}
```

Comment:
The throws RuntimeException is not needed here. Also would it be possible to reflow the comment to avoid the wildly long line.